benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/time_step_mod.f90

```diff
@@ -932,7 +882,9 @@ SUBROUTINE psy_init()
         IF (.not. initialised) THEN
             initialised = .True.
             ! Initialise the OpenCL environment/device
```

Comment:
So we always run on device 1? I thought this was configurable via an environment variable to FortCL? What am I missing?

Reply:
Note that this is the device number for a given platform. The environment variable lets us choose the platform: host CPU, Nvidia GPU, Xilinx FPGA, ... , the device number chooses if there are multiple devices (e.g. 2 GPU on Glados) where to run them. This was already 1 by default, I think in this implementation I just made explicit the optional values.

PSyclone-generated OpenCL uses a round-robin using the MPI rank.

Reply:
Thanks for explaining.